--- Makefile~	2009-01-04 19:31:32.000000000 +0100
+++ Makefile	2009-01-04 19:32:40.000000000 +0100
@@ -55,8 +55,8 @@
 CFLAGS += -W -Wall -Wmissing-declarations -Wwrite-strings
 CFLAGS +=  $(shell libgcrypt-config --cflags)
 CPPFLAGS += -DVERSION=\"$(VERSION)\" $(OPENSSL_GPL_VIOLATION)
-LDFLAGS ?= -g
-LDFLAGS += $(shell libgcrypt-config --libs) $(OPENSSLLIBS)
+LFLAGS ?= -g
+LDFLAGS += $(LFLAGS) $(shell libgcrypt-config --libs) $(OPENSSLLIBS)
 
 ifeq ($(shell uname -s), SunOS)
 LDFLAGS += -lnsl -lresolv -lsocket
